events/sink.go

```diff
@@ -0,0 +1,59 @@
+package events
+
+import (
+	"context"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/containerd/containerd/api/types/event"
+	"github.com/containerd/containerd/namespaces"
+	goevents "github.com/docker/go-events"
+	"github.com/pkg/errors"
+)
+
+type sinkEvent struct {
+	ctx   context.Context
+	event Event
+}
+
+type eventSink struct {
```

Comment:
Sounds okay. I don't see this `Close` method being called currently. Maybe we should remove it for now to avoid a possible "write to closed channel" panic.